src/handler.js

```diff
@@ -5,24 +5,70 @@ require('dotenv').config();
 const jf = require('jsonfile');
 
 function processMessage(event) {
-  const message = event.body
+  let message = event.body;
+  message = JSON.parse(message);
   return message;
 }
 
 function entry(event, context, callback) {
   const bit = new BitmovinTransmuxingLambda();
   console.log('Starting new bitmovin transmuxing job');
   const message = processMessage(event);
-  bit.createTransmuxes(message);
-  callback(null, 'Success!');
+  bit.start(message)
+    .then((res) => {
+      console.log('Successfully created and transfered transmuxings', res);
+      let message = {
+        statusCode: 200,
+        headers: {
+          "Content-Type": "application/json"
+        },
+        body: "Success, "
+      }
+      callback(null, message);
+    })
+    .catch((error) => {
+      console.log("ERROR failed to create/transfer transmuxing to S3", error);
+      let message = {
+        statusCode: 500,
+        headers: {
+          "Content-Type": "application/json"
+        },
+        body: "Error, " + JSON.stringify(error)
+      }
+      callback(null, message);
+    }
+  );
 }
 
 function test() {
   const bit = new BitmovinTransmuxingLambda();
   console.log('Starting new bitmovin transmuxing job');
   const event = jf.readFileSync('event.json');
   const message = processMessage(event);
-  bit.createTransmuxes(message);
+  bit.start(message)
```

Comment:
i can live with this for now, in the long terms would be great to figure out how the test method can also do a callback :)